### PR TITLE
fix(naming): fix default user email in README.md

### LINKED ISSUE #1870: doc: default account for user email
Hi guys,

the default account is `user@example.com` as the original user `user` was gone in this commit (https://github.com/kubeflow/manifests/commit/c5cb42f02f8513259ed19e41977b53e838c0d2e5)
```
kubectl describe configmap dex -n auth
...
staticPasswords:
- email: user@example.com
  hash: $2y$12$4K/VkmDd1q1Orb3xAt82zu8gk7Ad6ReFR4LCP9UeYE90NLiN9Df72
  # https://github.com/dexidp/dex/pull/1601/commits
  # FIXME: Use hashFromEnv instead
  username: user
  userID: "15841185641784"

```

however, [README.md](https://github.com/kubeflow/manifests/blob/master/README.md#installation) still use `user`:
```
In both options, we use a default username (user) and password (12341234).
```

### PR DESCRIPTION
**Pull request by hsinhoyeh**
Fixes https://github.com/kubeflow/manifests/issues/1870

**Which issue is resolved by this Pull Request:**
Resolves #1870

**Description of your changes:**
Just update `READMD.md` to reflect default user name to a valid email address

**Checklist:**
- [x] Unit tests pass:
  **Make sure you have installed kustomize == 3.2.1**
    1. `make generate-changed-only`
    2. `make test`

### CODE REVIEW

**Comment by hsinhoyeh** [comment]
just tagged @DavidSpek @yanniszark as I saw your previous discussion and maybe you have some inputs. thanks

File: README.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ kustomize build common/istio-1-9-0/istio-install/base | kubectl apply -f -
 
 #### Dex
 
-Dex is an OpenID Connect Identity (OIDC) with multiple authentication backends. In this default installation, it includes a static user named `user`. By default, the user's password is `12341234`. For any production Kubeflow deployment, you should change the default password by following [the relevant section](#change-default-user-password).
+Dex is an OpenID Connect Identity (OIDC) with multiple authentication backends. In this default installation, it includes a static user named `user` with E-mail `user@example.com`. By default, the user's password is `12341234`. For any production Kubeflow deployment, you should change the default password by following [the relevant section](#change-default-user-password).
```
Comment (yanniszark) [inline README.md:139]: I think we can only include email here. Makes this less confusing for users:

Old:

it includes a static user named `user` with E-mail `user@example.com`.

New:
it includes a static user named `user@example.com`.

File: README.md [hunk 2]
```diff
@@ -361,7 +361,7 @@ kustomize build apps/xgboost-job/upstream/overlays/kubeflow | kubectl apply -f -
 
 #### User Namespace
 
-Finally, create a new namespace for the the default user (named `user`).
+Finally, create a new namespace for the the default user (named `user-example-com`).
```
Comment (yanniszark) [inline README.md:364]: The namespace name is `kubeflow-user-example-com`, not `user-example-com`. How about we include both the user name and the namespace name?

Old:

Finally, create a new namespace for the the default user (named `user-example-com`).

New: 

Finally, create a new namespace (named `kubeflow-user-example-com`) for the the default user (named `user@example.com`).

**Review by yanniszark — CHANGES_REQUESTED** [review]
@hsinhoyeh thanks a lot for this PR! I have a couple of nits and we should be good to go.

File: README.md (hunk 1 above)
Comment (hsinhoyeh) [reply README.md]: done

**Comment by hsinhoyeh** [comment]
> @hsinhoyeh thanks a lot for this PR! I have a couple of nits and we should be good to go.

all done. please check

File: README.md [hunk 3]
```diff
@@ -361,7 +361,7 @@ kustomize build apps/xgboost-job/upstream/overlays/kubeflow | kubectl apply -f -
 
 #### User Namespace
 
-Finally, create a new namespace for the the default user (named `user`).
+Finally, create a new namespace (named `kubeflow-user-example-com`) for the the default user (named `user@example.com`).
```
Comment (mkbhanda) [inline README.md:364]: s/the the/the

duplicate the

**Review by mkbhanda — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (hsinhoyeh) [reply README.md]: done

**Comment by hsinhoyeh** [comment]
@yanniszark @mkbhanda sorry to pin you here, but I think this PR is good to go now?

**Pull request closed without merging**